### DISCUSSION TITLE
RBAC with Directus

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#15987] [Archive]
Hello, I am new to Directus and I would like to implement a special RBAC logic. 

In a collection I define the relational fields "owner" and "reader". Now the role "AuthUser" should ensure that only the users in the fields are authorized (owner = read/write, reader = read only). 

This can be achieved with the filters, but I can't find the right setting in the UI, so that the role "AuthUser" works correctly.

![image](https://user-images.githubusercontent.com/48225313/195628938-5125a299-4cae-4904-adfa-b2691810faa5.png)

**Comment by connorwinston** [comment]
Can you clarify a bit on how you have your relations set up for the owner/reader and then is there just a single directus_role called AuthUser?

**Reply by connorwinston** [reply]
Here is an example of an Owner (M2O) and Readers (M2M) set up and how you would configure it so they can both read their related items.
<img width="592" alt="image" src="https://user-images.githubusercontent.com/12714889/195658428-580fb5cf-7893-4d6b-aba3-a78bf50c3930.png">

Then if you just want the owner to be able to edit:
<img width="598" alt="image" src="https://user-images.githubusercontent.com/12714889/195658556-ad963d93-d5af-4df7-a98b-a0ddc6b64b68.png">

**Reply by ghost** [reply]
Thank you .. and its´s working.
